cardano-testnet/src/Testnet/Start/Cardano.hs

```diff
@@ -349,22 +366,20 @@ cardanoTestnet
 
   let tempBaseAbsPath = makeTmpBaseAbsPath $ TmpAbsolutePath tmpAbsPath
 
-  node1sprocket <- H.headM $ testnetSprockets runtime
+  node1sprocket <- case uncons $ testnetSprockets runtime of
+        Just (sprocket, _) -> pure sprocket
+        Nothing            -> throwString "No testnet sprocket available"
```

Comment:
```suggestion
  node1sprocket <- case testnetSprockets runtime of
        sprocket:_ -> pure sprocket
        []         -> throwString "No testnet sprocket available"
```
Because it is a list we can just use pattern matching